MMM-ArduPort.js

```diff
@@ -247,14 +247,15 @@ Module.register("MMM-ArduPort", {
                     errorTxt.innerHTML = "Error";
                     row.appendChild(errorTxt);
                 } else {
-                    row.appendChild(this.formatValue(sensor.value, sensor));
-
+                    var val = document.createElement("span");
+                    val.classList.add("port-data");
+                    
+                    if(sensor.name == "LM35") {val.innerHTML = sensor.value + " °C";}
+                    if(sensor.name == "MQ2") {val.innerHTML = sensor.value + " %";}
+                    if(sensor.name == "HCSR04") {val.innerHTML = sensor.value + " cm";}
```

Comment:
You don't need to hard-code the formats here. We have _format_ options, see here: https://github.com/Dentrax/MMM-ArduPort#configuration-options

Reply:
hardcoding this is what fixed this issue since 
formatValue: function(value, sensor) {
is not working properly ( value remains undefined )

Reply:
Oh, got it. Instead of applying a workaround, have you enough free time to find the root cause and apply the fix in the correct place? I would be very grateful if you could give a hand for this. 🙏

Reply:
> Oh, got it. Instead of applying a workaround, have you enough free time to find the root cause and apply the fix in the correct place? I would be very grateful if you could give a hand for this. 🙏

i will try to fix , but i need todo more research , fairly new to this type of coding.
but i am familiar with python and js , but more in bot developing